### PR TITLE
add designers as approved codeowners for docs

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

This would allow for designers to independently approve and merge documentation changes

### CODE REVIEW

File: .github/CODEOWNERS
Comment (alex-ju) [inline .github/CODEOWNERS:5]: meaning that any changes to files within this directory will require approval from members of either of these teams

**Review by alex-ju — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (Dhaulagiri) [reply .github/CODEOWNERS:5]: is it "either" or "both"? I believe it's both _can_ but only 1 is required to unblock merging?

Comment (didoo) [inline .github/CODEOWNERS:5]: should we include also the `website/public/assets` folder? usually they add images for the documentation there, too

Comment (Dhaulagiri) [reply .github/CODEOWNERS]: good call. I added that as well.

File: .github/CODEOWNERS
Comment (shleewhite) [inline .github/CODEOWNERS:5]: issue: I think docs and assets need to be done on separate lines, github says its an invalid owner
Screenshot 2024-11-19 at 10 26 07 AM

Comment (Dhaulagiri) [reply .github/CODEOWNERS:5]: 🤦

Comment (alex-ju) [reply .github/CODEOWNERS]: I was about to leave the same comment – it's a shame you can't chain them

**Pull request merged**
